docs/topics/rn-resolved-issues.adoc

```diff
@@ -7,36 +7,24 @@
 
 At the time of the release the following resolved issues have been identified as major issues worth highlighting.
 
-For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12360214[{ProductVersion} resolved issues] in Jira.
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12364384[{ProductVersion} resolved issues] in Jira.
 
 .Major resolved issues
-[cols="25%,75%",options="header"]
+[cols="25%,25%,50%",options="header"]
 |====
 |ID
+|Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUP-2941[WINDUP-2941]
-|MTA reports load slowly, do not include Target Runtime labels on the *Application List* screen, and do not have working feedback controls.
+|link:https://issues.redhat.com/browse/WINDUP-3082[WINDUP-3082]
+|Eclipse plugin
+|CodeReady Studio and Eclipse IDE: A `NullPointerException` error occurs when browsing rules in the *Rulesets* view.
 
-|link:https://issues.redhat.com/browse/WINDUP-2333[WINDUP-2333]
-|The {WebName} installed on OpenShift 4.4 and earlier versions cannot access the generated reports over an HTTPS connection.
-
-|link:https://issues.redhat.com/browse/WINDUP-2801[WINDUP-2801]
-|The {WebName} installed on OpenShift 4.5 and later versions cannot access the generated reports over an HTTPS connection.
-
-|link:https://issues.redhat.com/browse/WINDUP-2910[WINDUP-2910]
-|Analysis of applications built with JDK 11 generates `Unparseable Class File` errors.
-
-|link:https://issues.redhat.com/browse/WINDUP-2759[WINDUP-2759]
-|Lucene index builder mistakenly identifies Spring packages as application packages.
-
-|link:https://issues.redhat.com/browse/WINDUP-2591[WINDUP-2591]
-|CSV report cannot be exported if the *Skip reports* option is enabled.
-
-|link:https://issues.redhat.com/browse/WINDUP-2755[WINDUP-2755]
-|IDE plugin 5.0.1 cannot run with Eclipse 2020-06 because a null pointer exception causes analyses to fail.
-
-|link:https://issues.redhat.com/browse/WINDUP-2841[WINDUP-2841]
-|VS Code does not recognize custom CLI path in MTA settings.
+|link:https://issues.redhat.com/browse/WINDUP-3083[WINDUP-3083]
+|Eclipse plugin
+|CodeReady Studio and Eclipse IDE: When you click or double-click an issue in the *Issue Explorer* pane, the *Issue Details* pane is blank.
```

Comment:
If you have Eclipse plugin in the component column, you don't need CRS and Eclipse in the Summary column.